test-no_std/ci.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -e
+
+# setup
+export RUSTFLAGS="-D warnings"
+cargo install --path ..
```

Comment:
We shouldn't have 2 CI scripts, the entire point is to have that one script do all the CI things.

Can you move this crate into `test/` and extend the existing script instead?

Reply:
Like this? :)

Reply:
Yes, thanks. :)

Let's see what CI says. Will this really work on the macOS and Windows builders?